emsdk_manifest.json

```diff
@@ -56,548 +38,108 @@
   },
-
-  {
-    "id": "node",
-    "version": "8.9.1",
-    "bitness": 32,
-    "arch": "x86",
-    "windows_url": "node-v8.9.1-win-x86.zip",
-    "linux_url": "node-v8.9.1-linux-x86.tar.xz",
-    "activated_path": "%installation_dir%/bin",
-    "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
-    "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%",
-    "is_old": true
-  },
-  {
-    "id": "node",
-    "version": "8.9.1",
-    "arch": "arm",
-    "bitness": 32,
-    "linux_url": "https://nodejs.org/dist/v8.9.1/node-v8.9.1-linux-armv7l.tar.xz",
-    "activated_path": "%installation_dir%/bin",
-    "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
-    "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%",
-    "is_old": true
-  },
-  {
-    "id": "node",
-    "version": "8.9.1",
-    "bitness": 64,
-    "arch": "x86_64",
-    "macos_url": "node-v8.9.1-darwin-x64.tar.gz",
-    "windows_url": "node-v8.9.1-win-x64.zip",
-    "linux_url": "node-v8.9.1-linux-x64.tar.xz",
-    "activated_path": "%installation_dir%/bin",
-    "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
-    "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%",
-    "is_old": true
-  },
-  {
-    "id": "node",
-    "version": "8.9.1",
-    "arch": "aarch64",
-    "bitness": 64,
-    "linux_url": "node-v8.9.1-linux-arm64.tar.xz",
-    "activated_path": "%installation_dir%/bin",
-    "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
-    "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%",
-    "is_old": true
-  },
-  {
-    "id": "node",
-    "version": "12.18.1",
-    "bitness": 32,
-    "arch": "x86",
-    "windows_url": "node-v12.18.1-win-x86.zip",
-    "activated_path": "%installation_dir%/bin",
-    "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
-    "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%",
-    "is_old": true
-  },
-  {
-    "id": "node",
-    "version": "12.18.1",
-    "arch": "arm",
-    "bitness": 32,
-    "linux_url": "node-v12.18.1-linux-armv7l.tar.xz",
-    "activated_path": "%installation_dir%/bin",
-    "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
-    "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%",
-    "is_old": true
-  },
-  {
-    "id": "node",
-    "version": "12.18.1",
-    "bitness": 64,
-    "arch": "x86_64",
-    "macos_url": "node-v12.18.1-darwin-x64.tar.gz",
-    "windows_url": "node-v12.18.1-win-x64.zip",
-    "linux_url": "node-v12.18.1-linux-x64.tar.xz",
-    "activated_path": "%installation_dir%/bin",
-    "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
-    "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%",
-    "is_old": true
-  },
   {
     "id": "node",
-    "version": "12.18.1",
-    "arch": "aarch64",
-    "bitness": 64,
-    "linux_url": "node-v12.18.1-linux-arm64.tar.xz",
+    "version": "8.0.0",
```

Comment:
What is the actual version of NodeJS we are going to ship with this ?

Reply:
![image](https://github.com/dotnet/emsdk/assets/271576/5f08daea-5e32-4576-a525-4943565080a7)

https://nodejs.dev/en/about/releases/

Reply:
> What is the actual version of NodeJS we are going to ship with this ?

v14.18.2 is the version bundled with emsdk 3.1.34. We can deviate from this if we feel like it (see also shipping an old weird Python version)